srcpkgs/chromium/template

```diff
@@ -160,6 +159,9 @@ do_configure() {
 		enable_widevine=true
 		enable_hangout_services_extension=true'
 
+	#  Jumbo/Unity builds: https://chromium.googlesource.com/chromium/src/+/master/docs/jumbo.md
+	conf+="use_jumbo_build=true"
+
```

Comment:
Not quite sure how beefy the builders are, jumbo builds can take up to 3-4GB per build job for linking. However, it's _way_ faster than building without jumbo building (jumbo building chromium takes about 1 1/2 hours vs. 6-8 hours without it on my Ryzen 1700) and upstream only tests with jumbo building - although building without jumbo building usually doesn't need lots of patches, in this case only https://github.com/void-linux/void-packages/pull/3880/files#diff-12a5c377aace0348303db474b2e9bafa

Reply:
@Duncaen

Reply:
I've optionalised it for now.

Reply:
If it's just 3-4 GB for builds, and not, say, 10, I think the relevant builders will do fine with jumbo builds.

Reply:
Alright, I'll enable it by default, I guess we'll see if that works out :)